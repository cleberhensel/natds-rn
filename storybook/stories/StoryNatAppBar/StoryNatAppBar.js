import React, { useState } from 'react';
import {
  View,
  ScrollView,
  SafeAreaView,
  Dimensions,
} from 'react-native';
import {
  NatContainer,
  NatAppBar,
  NatLogo,
} from '../../../lib';

const styles = {
  defaultScreen: {
    flex: 1,
    width: Dimensions.get('window').width,
  },
};

export const StoryNatAppBar = () => {
  const logo = () => (<NatLogo name="logo-natura-horizontal" width={150} height={56} />);

  return (
    <SafeAreaView style={styles.defaultScreen}>
      <NatContainer>
        <ScrollView>
          <NatAppBar
            title="Center title"
            alignTitle="center"
            iconLeft={{ svg: { name: 'outlined-navigation-menu' } }}
            iconRight={{ svg: { name: 'outlined-action-autofilter' } }}
            onPressLeft={() => alert('press')}
            onPressRight={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Title Text"
            iconLeft={{ svg: { name: 'outlined-navigation-menu' } }}
            iconRight={{ svg: { name: 'outlined-action-autofilter' } }}
            onPressLeft={() => alert('press')}
            onPressRight={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Title Text"
            iconLeft={{ svg: { name: 'outlined-navigation-menu' } }}
            onPressLeft={() => alert('press')}
            onPress={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Title Text"
            onPress={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Title Text"
            node={logo()}
            iconLeft={{ svg: { name: 'outlined-navigation-menu' } }}
            iconRight={{ svg: { name: 'outlined-action-autofilter' } }}
            onPressLeft={() => alert('press')}
            onPressRight={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Title Text"
            alignTitle="center"
            node={logo()}
            iconLeft={{ svg: { name: 'outlined-navigation-menu' } }}
            onPressLeft={() => alert('press')}
            onPress={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Title Text"
            node={logo()}
            onPress={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Exclusive"
            showIconLeft={false}
            iconRight={{ svg: { name: 'outlined-action-autofilter' } }}
            onPressLeft={() => alert('press')}
            onPressRight={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Exclusive"
            iconLeft={{ svg: { name: 'outlined-navigation-menu' } }}
            showIconRight={false}
            onPressLeft={() => alert('press')}
            onPress={() => alert('press')}
          />
          <View style={{ height: 10 }} />
          <NatAppBar
            title="Exclusive"
            showIconLeft={false}
            showIconRight={false}
            onPress={() => alert('press')}
          />
        </ScrollView>
      </NatContainer>
    </SafeAreaView>
  );
};

StoryNatAppBar.propTypes = {
};
