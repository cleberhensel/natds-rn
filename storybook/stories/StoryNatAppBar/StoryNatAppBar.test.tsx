import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components/native';
import { StoryNatAppBar } from './StoryNatAppBar';

describe('StoryNatAppBar component', () => {
  it('Should render StoryNatAppBar component ', () => {
    const button = renderer.create(<StoryNatAppBar />).toJSON();

    expect(button).toMatchSnapshot('StoryNatAppBar snapshot');
  });
});
