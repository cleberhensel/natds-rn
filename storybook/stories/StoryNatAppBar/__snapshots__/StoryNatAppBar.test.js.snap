// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StoryNatAppBar component Should render StoryNatAppBar component  1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#FFFFFF",
          "bottom": 0,
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "left": 0,
          "paddingBottom": 0,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 0,
          "right": 0,
          "top": 0,
        },
        Object {
          "backgroundColor": "#FAFAFA",
          "marginTop": 0,
          "width": 750,
        },
      ]
    }
  >
    <RCTScrollView>
      <View>
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={Object {}}
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 16,
                  "paddingRight": 0,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            >
              <SvgMock
                height={24}
                style={
                  Object {
                    "fill": "#333333",
                  }
                }
                width={24}
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 40,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessibilityLabel="Title Text"
              style={
                Array [
                  Object {
                    "color": "#333333",
                    "fontSize": 20,
                    "marginLeft": 0,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingTop": 16,
                  },
                ]
              }
            >
              Title Text
            </Text>
          </View>
          <View
            style={
              Object {
                "position": "absolute",
                "right": 0,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 0,
                  "paddingRight": 16,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            >
              <SvgMock
                height={24}
                style={
                  Object {
                    "fill": "#333333",
                  }
                }
                width={24}
              />
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={Object {}}
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 16,
                  "paddingRight": 0,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            >
              <SvgMock
                height={24}
                style={
                  Object {
                    "fill": "#333333",
                  }
                }
                width={24}
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 40,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessibilityLabel="Title Text"
              style={
                Array [
                  Object {
                    "color": "#333333",
                    "fontSize": 20,
                    "marginLeft": 0,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingTop": 16,
                  },
                ]
              }
            >
              Title Text
            </Text>
          </View>
          <View
            style={
              Object {
                "position": "absolute",
                "right": 0,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 0,
                  "paddingRight": 16,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={Object {}}
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 16,
                  "paddingRight": 0,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 40,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessibilityLabel="Title Text"
              style={
                Array [
                  Object {
                    "color": "#333333",
                    "fontSize": 20,
                    "marginLeft": 0,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingTop": 16,
                  },
                ]
              }
            >
              Title Text
            </Text>
          </View>
          <View
            style={
              Object {
                "position": "absolute",
                "right": 0,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 0,
                  "paddingRight": 16,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={Object {}}
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 16,
                  "paddingRight": 0,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            >
              <SvgMock
                height={24}
                style={
                  Object {
                    "fill": "#333333",
                  }
                }
                width={24}
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 40,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <SvgMock
              width={200}
            />
          </View>
          <View
            style={
              Object {
                "position": "absolute",
                "right": 0,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 0,
                  "paddingRight": 16,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            >
              <SvgMock
                height={24}
                style={
                  Object {
                    "fill": "#333333",
                  }
                }
                width={24}
              />
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={Object {}}
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 16,
                  "paddingRight": 0,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            >
              <SvgMock
                height={24}
                style={
                  Object {
                    "fill": "#333333",
                  }
                }
                width={24}
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 40,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <SvgMock
              width={200}
            />
          </View>
          <View
            style={
              Object {
                "position": "absolute",
                "right": 0,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 0,
                  "paddingRight": 16,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={Object {}}
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 16,
                  "paddingRight": 0,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            />
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 40,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <SvgMock
              width={200}
            />
          </View>
          <View
            style={
              Object {
                "position": "absolute",
                "right": 0,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 0,
                  "paddingRight": 16,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            />
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessibilityLabel="Exclusive"
              style={
                Array [
                  Object {
                    "color": "#333333",
                    "fontSize": 20,
                    "marginLeft": 0,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingTop": 16,
                  },
                ]
              }
            >
              Exclusive
            </Text>
          </View>
          <View
            style={
              Object {
                "position": "absolute",
                "right": 0,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 0,
                  "paddingRight": 16,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            >
              <SvgMock
                height={24}
                style={
                  Object {
                    "fill": "#333333",
                  }
                }
                width={24}
              />
            </View>
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={Object {}}
          >
            <View
              accessible={true}
              focusable={true}
              isTVSelectable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "alignSelf": "center",
                  "flexBasis": 0,
                  "flexGrow": 1,
                  "flexShrink": 1,
                  "height": 56,
                  "justifyContent": "center",
                  "opacity": 1,
                  "paddingBottom": 16,
                  "paddingLeft": 16,
                  "paddingRight": 0,
                  "paddingTop": 16,
                  "width": 40,
                }
              }
            >
              <SvgMock
                height={24}
                style={
                  Object {
                    "fill": "#333333",
                  }
                }
                width={24}
              />
            </View>
          </View>
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 40,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessibilityLabel="Exclusive"
              style={
                Array [
                  Object {
                    "color": "#333333",
                    "fontSize": 20,
                    "marginLeft": 0,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingTop": 16,
                  },
                ]
              }
            >
              Exclusive
            </Text>
          </View>
        </View>
        <View
          style={
            Object {
              "height": 10,
            }
          }
        />
        <View
          accessibilityRole="header"
          accessible={true}
          style={
            Array [
              Object {
                "alignItems": "stretch",
                "backgroundColor": "#FFFFFF",
                "flexBasis": 0,
                "flexDirection": "row",
                "flexGrow": 1,
                "flexShrink": 1,
                "height": 56,
                "paddingBottom": 0,
                "paddingLeft": 0,
                "paddingRight": 0,
                "paddingTop": 0,
                "position": "relative",
                "width": "100%",
              },
              Object {
                "elevation": 1,
                "shadowColor": "#000",
                "shadowOffset": Object {
                  "height": 2,
                  "width": 0,
                },
                "shadowOpacity": 0.24,
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "flex-start",
                "bottom": 0,
                "justifyContent": "flex-start",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <Text
              accessibilityLabel="Exclusive"
              style={
                Array [
                  Object {
                    "color": "#333333",
                    "fontSize": 20,
                    "marginLeft": 0,
                    "paddingBottom": 16,
                    "paddingLeft": 16,
                    "paddingTop": 16,
                  },
                ]
              }
            >
              Exclusive
            </Text>
          </View>
        </View>
      </View>
    </RCTScrollView>
  </View>
</RCTSafeAreaView>
`;
